src/egon/data/datasets/gas_grid.py

```diff
@@ -213,17 +215,103 @@ def insert_CH4_nodes_list(gas_nodes_list):
     )
 
 
-def insert_gas_pipeline_list(gas_nodes_list):
+def insert_gas_buses_abroad(scn_name="eGon2035"):
+    """Insert central gas buses in foreign countries to db, same buses than the foreign AC buses
+    Parameters
+    ----------
+    scn_name : str
+        Name of the scenario
+
+    Returns
+    -------
+    gdf_abroad_buses : dataframe
+        Dataframe containing the gas in the neighbouring countries and one in the center of Germany in test mode
+    """
+    main_gas_carrier = scenario_parameters.gas(scn_name)["main_gas_carrier"]

```

Comment:
@AmeliaNadal @ClaraBuettner 
To be consistent, shouldn't the scenario parameters read by using [`egon.data.datasets.scenario_parameters.get_sector_parameters()`](https://github.com/openego/eGon-data/blob/0730acfae19def9381f184b28f3304c5f7c0be3c/src/egon/data/datasets/scenario_parameters/__init__.py#L101)?

Reply:
Thanks for the remarks, I made the requiered change.